vassal-app/src/main/java/VASSAL/build/GpIdChecker.java

```diff
@@ -44,8 +46,8 @@
   protected int maxId;
   protected boolean useName = false;
   protected boolean extensionsLoaded = false;
-  final HashMap<String, SlotElement> goodSlots = new HashMap<>();
-  final ArrayList<SlotElement> errorSlots = new ArrayList<>();
+  final Map<String, SlotElement> goodSlots = new HashMap<>();
+  final List<SlotElement> errorSlots = new ArrayList<>();
```

Comment:
Woah, these ones are package private? These should probably be either private or protected.

Reply:
I'll leave you to decide which

Reply:
Make them private. This class doesn't make sense to extend anyhow.